insert_data_to_vector_db_weaviate/main.bal

```diff
@@ -0,0 +1,64 @@
+import ballerina/io;
+import ballerinax/googleapis.sheets;
+import ballerinax/openai.embeddings;
+import ballerinax/weaviate;
+
+configurable string openAIToken = ?;
+configurable string weaviateToken = ?;
+configurable string weaviateURL = ?;
+
+configurable string sheetAccessToken = ?;
+configurable string sheetId = ?;
+configurable string sheetName = ?;
+
+const RANGE = "A2:C630";
+const NO_OF_COLUMNS = 3;
+const className = "QuestionAnswerStore";
+const MODEL = "text-embedding-ada-002";
+
+final embeddings:Client openai = check new ({auth: {token: openAIToken}});
+final weaviate:Client weaviate = check new ({auth: {token: weaviateToken}, timeout: 100}, weaviateURL);
+final sheets:Client gsheets = check new ({auth: {token: sheetAccessToken}});
```

Comment:
Since this is main function, no need to define clients outside of the main. Instead move it closer to the first invocation line.

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/60/commits/2bbad1aa77295f40a3191110f4502e51165dd282.